jsk_apc2016_common/config/bin_shelf.yaml

```diff
@@ -0,0 +1,14 @@
+initial_pos_list_bottom: [[-0.27, 0.25, -0.41], [-0.24, -0.01, -0.41], [-0.24, -0.32, -0.41], [-0.27, 0.25, -0.12], [-0.24, -0.01, -0.12], [-0.24, -0.32, -0.12]]
+initial_rot_list_bottom: [[0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]
```

Comment:
I don't care about 2015 or 2016. That depends on which you use to run the demo for the current milestone.
If you stop using launch files in jsk_2015_05_baxter_apc, it is time to copy or add extended launch file in jsk_2016_01_baxter_apc.